### ISSUE TITLE
Math Notation Font Size Bug and Unclear Effects

### ISSUE THREAD

**Issue by athai-009** [Sinacs/Joplin.Plugin.ReMoods.Theme#43]
## Environment:

 - OS: Windows 11
 - Joplin version: 2.10.15
 - ReMoods theme version: 4.3.0

## Describe the bug:

When adjusting "Render Viewer - Math Notation Font Size (px)", the results are unclear and depend on if the math hover zoom effect is enabled. If hover zoom is enabled, I think increasing the size causes math to shrink on hover instead of zoom. If hover zoom is disabled, it force-enables hover zoom and makes pre-zoomed text small. I honestly don't understand the setting, but I don't think it was intended to force-enable hover zoom.

## To Reproduce:

1. Navigate to ReMoods Theme plugin settings
2. Uncheck "Render Viewer - * Enable math notation hover zoom effect"
3. Set "Render Viewer - Math Notation Font Size (px)" to 2
4. Click apply
5. Click back
6. Restart Joplin
7. View any math output in the render viewer

## Expected behavior:

I expected to set "Render Viewer - Math Notation Font Size (px)" to 2 and have math notation to use a font size 2 pts bigger than the base font size (or 2 times).

**Comment by Sinacs** [comment]
I'm not sure if I do understand the problem you met, but I did find some issues about it:

1. Currently, the font size value of "Enable math notation hover zoom effect" is incorrectly associated with the "Base Font Size", so when the "Math Symbol Font Size" setting is greater than 1.5 times the `basic font size`, it will cause the font size to shrink when the effect is triggered.

2. The setting property name "Enable math notation hover zoom effect" does not clearly describe the effect of the setting.

---

So, I would solve the above issues as follows:

1. Change the property name from `Enable math notation hover zoom effect` to `Enable math notation auto-enlarged effect'. 

2. Make sure the effect should be 1.5 times larger than the `Math notation font size` when the mouse hovers over the math notation.

---

Do you think the above fixes will solve the problem you met? Please let me know if there is anything missing in my understanding.

P.S. Please make sure you only input the numbers as any font size values, because all font size values have a default unit which is shown at the end of the property name. e.g. `Math notation font size (px)`, for more reference you can look around the [Settings Introduction](#https://github.com/Sinacs/Joplin.Plugin.ReMoods.Theme/blob/master/Doc-SettingsIntroduction.md).

**Comment by athai-009** [comment]
Sorry for the delayed response. I've been too busy.

---

1. That is a more fitting name. 
2. I tried a basic font size of 30 and a math font size of 44 (less than 1.5 times), and it does correctly enlarge on hover but only by a little (not by 15 px).

---

Right under the setting, you have text that says "0: Equal to Base Font Size" which made me think the setting is additive. E.g., if basic font size was 16 and the math font size setting was 4, the actual math font size should be 20. What's unclear was whether I was setting font size directly or relative to the basic font size (whether additively or multiplicative) based on the results I got.

Based on what you've said, I'm guessing it's directly font size, but 0 is a special case. It's still unclear to me what it does since I tested the case mentioned above.

What is clear though is that when the **hover zoom effect is disabled**, non-zero values of math font size re-enables it.
